Documentation/guideline/csharp_coding_standards.md

```diff
@@ -7,7 +7,7 @@ C# Coding Standards
 
 Introduction
 ----------------
-The coding standard will be used in conjunction with customized version of *StyleCop* and *FxCop* [**TODO**] during both development and build process. This will help ensure that the standard is followed by all developers on the team in a consistent manner.
+The coding C# Coding Standards will be used in conjunction with customized versions of *StyleCop* and *FxCop* [**TODO**] during both development and build process. This will help ensure that all developers on the team in a consistent manner.
```

Comment:
*This will help ensure that all developers on the team in a consistent manner* => syntax error?